src/whatlang.h

```diff
@@ -5,6 +5,13 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+typedef enum whatlang_status : uint8_t {
+    OK = 0,
```

Comment:
These enum values need prefixing. We should also switch the constants over to being enums too.

Reply:
Improved enum representation now. Users can choose between const definitions or clang / C++11 compatible uint8_t enums.